### ISSUE TITLE
GRCh38 v GRCh37 in UKB

### ISSUE THREAD

**Issue by asoroosh** [totajuliusd/topr#17]
Hello, 

Thank you for the amazing work and the package. 

When using UKB GWAS results, I use GRCh37 (based on https://biobank.ndph.ox.ac.uk/showcase/label.cgi?id=263). However, The plots generated using `topr` using GRCh37 is not aligned with the gene I query from `topr`. 

Please see the following examples:

As an example, when I use `regionplot` to locate TOMM40 using 37 genome build

``` 
topr::regionplot(gg_all, gene = 'TOMM40', build = '37')
[1] "Zoomed to region:  chr19:44790568-45003689"  
```

![chr19_37](https://github.com/totajuliusd/topr/assets/8735002/81350c82-b172-43cd-b66e-8dc6d4bbafed)

Here is an example of using exactly same data, but with build 38

``` 
topr::regionplot(gg_all, gene = 'TOMM40', build = '38')
[1] "Zoomed to region:  chr19:44790568-45003689"  
```

![chr19_38](https://github.com/totajuliusd/topr/assets/8735002/5c286956-ac50-47a0-b292-4eb403241bd1)

Thanks again for your package,

**Comment by totajuliusd** [comment]
Hi and thank you for telling me about this issue, this is clearly an error that needs fixing.  Although you ask for build 37, topr still pulls out the build 38 coordinates for gene TOMM40.  

I will fix this and release a new version of topr asap. In the meantime, you can create the regionplot centered on TOMM40 in build 37 using the **region** argument instead of the **gene** argument, like so:

topr::regionplot(gg_all, **region**="chr19:45293825-45506946", build = '37')

The regionplot centers in on the gene with an additional 100 kb extending in each direction.

You can get the gene coordinates for build 37 with the _get_gene_ command (ignore the Warning message that appears)

> topr::get_gene("TOMM40", build="37")
chrom gene_start gene_end gene_symbol        biotype
1 chr19   45393825 45406946      TOMM40 protein_coding

Thanks again for pointing this out so this can be fixed.
Tota

**Comment by totajuliusd** [comment]
HI again, this has been fixed in topr version 1.1.9 which is on its way to CRAN and should be available from there in one or two hours.
